Woof/WoofTests/Model/Preferences/PreferencesHandlerSaveTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+
+final class PreferencesHandlerSaveTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        PreferencesHandlerTestHelper.cleanStorage()
```

Comment:
I don't understand the intention of code. Please improve readability. I'd expect smth like:

```suggestion
        UserPreferencesStorage.removeAll()
```

Reply:
ok, I'll fix it

Reply:
fixed